packages/content/src/requests/items/formField/settings/StringFormFieldSettings.ts

```diff
@@ -0,0 +1,38 @@
+import { serialize, type, validate } from "@js-soft/ts-serval";
+import { FormFieldSettings, FormFieldSettingsJSON, IFormFieldSettings } from "./FormFieldSettings";
+
+export interface StringFormFieldSettingsJSON extends FormFieldSettingsJSON {
+    "@type": "StringFormFieldSettings";
+    allowNewLines?: true;
```

Comment:
actually newline is one word. So this should be `allowNewlines`. Same everywhere else.